public/video-ui/src/components/FormFields/TagPicker.js

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import ContentApi from '../../services/capi';
 import { tagsFromStringList, tagsToStringList } from '../../util/tagParsers';
-import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
 import { keyCodes } from '../../constants/keyCodes';
 import UserActions from '../../constants/UserActions';
 import TagTypes from '../../constants/TagTypes';
+import getTagDisplayNames from '../../util/getTagDisplayNames';
+import TextInputTagPicker from './TextInputTagPicker';
+import PureTagPicker from './PureTagPicker';
+import TagFieldValue from '../Tags/TagFieldValue';
+import CapiUnavailable from '../CapiSearch/CapiUnavailable';
 import DragSortableList from 'react-drag-sortable';
+import removeTagDuplicates from '../../util/removeTagDuplicates';
+import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
 
 export default class TagPicker extends React.Component {
+
   state = {
     capiTags: [],
-    inputString: '',
-    lastAction: UserActions.other,
     tagValue: [],
-    capiUnavailable: false
+    capiUnavailable: false,
+    showTags: true,
+    tagsVisible: false,
```

Comment:
from what I understood from the code, `showTags` is always the inverse of `tagsVisible` which made me think it could be one boolean value?